moe.emmaexe.ntfyDesktop.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "moe.emmaexe.ntfyDesktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "ntfyDesktop",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher"
+    ],
+    "modules": [
+        {
+            "name": "ntfyDesktop",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DND_BUILD_TYPE=Flatpak"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/emmaexe/ntfyDesktop/archive/refs/tags/v1.2.0.tar.gz",
+                    "sha256": ""
```

Comment:
sha25 is required.

Reply:
I will add it later once v1.2.0 releases.